pkgs/native_assets_builder/test_data/treeshaking_native_libs/ffigen.yaml

```diff
@@ -0,0 +1,22 @@
+# Run with `flutter pub run ffigen --config ffigen.yaml`.
+name: NativeCalcBindings
+description: |
+  Bindings for `src/native_add.h` and `src/native_multiply.h`.
+
+  Regenerate bindings with `flutter pub run ffigen --config ffigen.yaml`.
+output: "lib/src/treeshaking_native_libs_bindings_generated.dart"
+headers:
+  entry-points:
+    - "src/native_add.h"
+    - "src/native_multiply.h"
+  include-directives:
+    - "src/native_add.h"
+    - "src/native_multiply.h"
+preamble: |
+  // Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
2024? ditto in the files? (Not sure what's the right copyright year for copy-pasted files though.)